app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion "25.0.3"
     defaultConfig {
         applicationId "com.kyphipraniti.fitnesstasks"
-        minSdkVersion 16
+        minSdkVersion 21
```

Comment:
why are we changing minSdkVersion ? lot of people are still on kitkat.

Reply:
agreed

Reply:
We'll have to add separate layout files for datePickerMode attribute in layout/fragment_calendar to support 16.